draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -311,118 +435,80 @@ and media access delays and link-layer retransmission lead to increased jitter
-down.  Therefore, particular attention should be paid to
-characterizing the areas where the proposed mechanism does not
-perform well.
+The criteria in {{evaluation-criteria}} will be evaluated in the
+following scenarios, unless the proposal specifically presents its use in a
+scenario. The community MAY allow a proposal to progress even if the criteria
+indicate an unsatisfactory result for these scenarios.
 
-: As an example from an Experimental RFC, performance in a range of
-environments is discussed in Section&nbsp;12 of {{?RFC3649}} (HighSpeed
-TCP) and Section&nbsp;9.7 of {{?RFC4782}} (Quick-Start).
+In general, measurements from internet-scale deployments will not expose the
+properties of operation in these scenarios, as they are statistically small.
 
-(5)
-: Protection Against Congestion Collapse
+## Internet of Things
 
-: The alternate congestion control mechanism should either stop
-sending when the packet drop rate exceeds some threshold
-{{?RFC3714}}, or should include some notion of "full backoff".  For
-"full backoff", at some point the algorithm would reduce the
-sending rate to one packet per round-trip time and then
-exponentially backoff the time between single packet
-transmissions if congestion persists.  Exactly when either "full
-backoff" or a pause in sending comes into play will be
-algorithm-specific.  However, as discussed in {{!RFC2914}}, this
-requirement is crucial to protect the network in times of extreme
-congestion.
+(TODO: Write this section)
 
-: If "full backoff" is used, this bullet does not require that the
-full backoff mechanism must be identical to that of TCP
-{{?RFC2988}}.  As an example, this bullet does not preclude full
-backoff mechanisms that would give flows with different round-
-trip times comparable caapcity during backoff.
+## Satellite
 
-(6)
-: Protection Against Bufferbloat
+Satellite links often have long delays {{?RFC2488}} and high delay bandwidth
+products{{?RFC3649}}.
 
-: The alternate congestion control mechanism should reduce its sending
-rate if the round trip time (RTT) significantly increases. Exactly how
-the algorithm reduces its sending rate is algorithm specific.
+## Misbehaving Nodes
 
-: Bufferbloat {{Bufferbloat}} refers to the building of long queues in
-the network. Many network routers are configured with very large buffers.
-If congestion starts happening, classic TCP congestion control algorithms
-{{!RFC5681}} will continue sending at a high rate until the buffer fills
-up completely and packet losses occur. Every connection going through
-that bottleneck will experience high latency.
-This is particularly bad for highly interactive applications like games,
-but it also affects routine web browsing and video playing.
+The proposal should explore how the alternate congestion control
+mechanism performs with misbehaving senders, receivers, or
+routers.  In addition, the proposal should explore how the
+alternate congestion control mechanism performs with outside
```

Comment:
/misbehaving/ is likely a very unhelpful term here. I am not sure we actually need to evaluate from incorrectly implemented routers?

Reply:
I changed misbehaving to non-compliant. Routers are discussed later in the section, so I'll keep it here. Apparently Quick Start thought it was important enough to consider?

Reply:
OK with that change: This I think QuickStart discuses this because it is a CC method that relies on router feedback.